tests/CsvIteratorTest.php

```diff
@@ -36,7 +36,7 @@ public function testOpenNotExistsFile()
 
     public function testParseColumns()
     {
-        $header = $this->getJsonFixture('header.json');
```

Comment:
all presented test changes will become useless after correct implementation of task